src/app/backend/dbServices/fhir-servers.ts

```diff
@@ -56,6 +56,10 @@ class FhirServerConfigService extends FhirServerConfigServiceInternal {
 
   static async getFhirServerNames(): Promise<string[]> {
     const configs = await super.getFhirServerConfigs();
+    // Sort so that the default server is always first
+    configs.sort((a, b) =>
+      b.defaultServer === true ? 1 : a.defaultServer === true ? -1 : 0,
+    );
```

Comment:
Noticing on dev startup that since the default isn't initialized any way outside of user input, that Public HAPI ends up being the default server on a fresh run of the app since it shows up first. We _could_ just ask devs to mark Aidbox as default manually each time, but in self interest wondering if we could come up with something else 😅 

https://github.com/user-attachments/assets/5b1c2204-a816-4dca-b919-a1a75afc06ad

Can we think of a way between the way we're returning the servers here and the way we're setting [the default on the frontend](https://github.com/CDCgov/dibbs-query-connector/pull/576/files#diff-8e846d6c8b80796aed5e0baf27e741a83008e6a3bee47373f46ddb3c3e0527ce) to still have Aidbox populate? Maybe by also alphabetizing the list as well?

Reply:
Alphabetizing is a great idea! Right now, all of the dev servers that populate as part of our migrations default to false for default server, and so the app "defaults" to using the first server in the list, whatever that might be. I think what makes most sense is to modify our aidbox seeding script to insert aidbox into the fhir servers table as the default. Does that sound right to you?

Reply:
yeah adding it to the seeding script sounds like the best option!

Reply:
Done!